www/js/app.js

```diff
@@ -119,6 +119,8 @@ angular.module('starter', ['ionic','ngMessages'])
      var indexedTeams=[];
      $scope.calllists='';
      $scope.UserId=window.localStorage.getItem("UserId"); //get the login userid
+     //initalize the service url
+     const BASE_URL_VALUE = "http://192.168.0.137/LH_Mobile_Backend"
```

Comment:
The _**BASE_URL_VALUE**_ is supposed to be a single global constant to the application, in your code, I saw that there are 4 occurrences of this value.
The main purpose of using this value is to make it easy to globally change the URL to impact all the application REST calls.
Please see the following example of how to do it correctly in Ionic & Angular:
http://ilikekillnerds.com/2014/11/constants-values-global-variables-in-angularjs-the-right-way/